docs/design/LAG-for-ports/domain-model.puml

```diff
@@ -0,0 +1,23 @@
+@startuml
+
+title LAGs Domain Model
+
+class LogicalPort {
```

Comment:
I'm not sure about name `LogicalPort` because we have several types of Logical ports (for example BFD port is logical)
Maybe add `Lag` in name? `LagLogicalPort` or `LagPort`?

Reply:
Reworked to `LagLogicalPort`.